bedrock/settings/base.py

```diff
@@ -543,6 +543,7 @@ def get_app_name(hostname):
 CLUSTER_NAME = config("CLUSTER_NAME", default="")
 ENABLE_HOSTNAME_MIDDLEWARE = config("ENABLE_HOSTNAME_MIDDLEWARE", default=str(bool(APP_NAME)), parser=bool)
 ENABLE_VARY_NOCACHE_MIDDLEWARE = config("ENABLE_VARY_NOCACHE_MIDDLEWARE", default="true", parser=bool)
+IS_PROD = APP_NAME == "bedrock-prod"
```

Comment:
```suggestion
IS_MOZILLA_ORG = APP_NAME == "bedrock-prod"
```

Reply:
I confirmed with brett that `APP_NAME == "bedrock-prod"` will be `True` for both pocket and mozorg. So I think maybe we can leave it? I added a comment.

Reply:
OK, sure thing.